app/javascript/styles/components/Spotlights.scss

```diff
@@ -60,32 +52,20 @@
 }
 
 .calendar-spotlight {
-  @include media(large) {
-    grid-template-columns: repeat(3, 1fr);
-    -ms-grid-columns: 1fr 1fr 1fr;
-    max-width: 870px;
-  }
-  @include media(medium) {
-    grid-template-columns: repeat(2, 1fr);
-    -ms-grid-columns: 1fr 1fr;
-    max-width: 560px;
-  }
-  @include media(small) { column-gap: 60px; }
-  @include media(x-small) { column-gap: 30px; }
-  align-self: center;
-  column-gap: 60px;
-  display: grid;
-  grid-template-columns: repeat(4, 1fr);
-  display: -ms-grid;
-  -ms-grid-columns: 1fr 1fr 1fr 1fr;
-  margin-bottom: 60px;
-  max-width: 1180px;
+  margin: 0 auto 60px;
+  max-width: 100%;
+  width: 1180px;
 
   &__image-wrapper {
-    grid-column: 1;
-    -ms-grid-column: 1;
+    display: inline-block;
     position: relative;
     width: 250px;
+
+    @media screen and (max-width: 1350px) {
+      padding: 4rem;
```

Comment:
Since this applies 4rem all around, it puts an extra 4rem of space between the image and the nav bar (which already has a 60px bottom margin). I also think that 1350px is too early for the poster to go full-screen; the calendar tiles still fit 4 to a row at this point, and the user has to scroll through an entire screen of poster before reaching any other content.